docker/common/Dockerfile

```diff
@@ -17,21 +17,28 @@ FROM toolchain AS builder
 COPY . /libra
 
 ENV RUSTFLAGS "-Ctarget-cpu=skylake -Ctarget-feature=+aes,+sse2,+sse4.1,+ssse3"
-RUN cargo build --release -p libra-node -p cli -p config-builder -p safety-rules && cd target/release && rm -r build deps incremental
+RUN cargo build --release -p libra-node -p cli -p config-builder -p libra-key-manager -p safety-rules && cd target/release && rm -r build deps incremental
 RUN strip /libra/target/release/config-builder
+RUN strip /libra/target/release/libra-key-manager
```

Comment:
Let's not strip key-manager and safety-rules for now, I think more useful stacktraces are worth it for now.